src/player.cpp

```diff
@@ -5456,6 +5460,24 @@ void player::suffer()
             }
         }
     }
+    for( const auto &m : my_mutations ) {
+        const mutation_branch &mdata = m.first.obj();
+        for( const body_part bp : all_body_parts ) {
+            if( calendar::once_every( 1_minutes ) ) {
+                const float wetness_percentage =  body_wetness[bp] / drench_capacity[bp]; // 0.0 - 1.0
+                const int dmg = mdata.weakness_to_water * wetness_percentage;
+                if( dmg > 0 ) {
+                    apply_damage( nullptr, bp, dmg );
+                    add_msg_player_or_npc( m_bad, _( "Your %s is damaged by the water." ),
+                                           _( "<npc_name>'s %s is damaged by the water." ), body_part_name( bp ) );
+                } else if( dmg < 0 && hp_cur[bp_to_hp( bp )] != hp_max[bp_to_hp( bp )] ) {
+                    heal( bp, abs( dmg ) );
+                    add_msg_player_or_npc( m_good, _( "Your %s is healed by the water." ),
+                                           _( "<npc_name>'s %s is healed by the water." ), body_part_name( bp ) );
```

Comment:
In messages "\<npcname\>" (no underscore) is replaced with the name of the npc. "\<npc_name\>" has no special meaning.